.nvidia-ci.yml

```diff
@@ -455,6 +527,9 @@ sign:ngc-ubuntu-rhel-rhcos:
     - .sign:ngc
   parallel:
     matrix:
+    - SIGN_JOB_NAME: ["ubuntu"]
+      VERSION: ["24.04"]
+      DRIVER_VERSION: ["535.216.03", "550.127.08"]
```

Comment:
I believe we can remove 535. I don't see Ubuntu 24.04 as a supported distribution in the latest R535 driver support matrix: https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-535-216-03/index.html#hardware-software-support

Reply:
corrected